src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktiviteterRepositoryV2.java

```diff
@@ -133,6 +149,19 @@ AND NOT (status = ANY (:ikkestatuser::varchar[]))
                 });
     }
 
+    private String hentSkjermeteDiskresjonskoder(boolean tilgangTilKode6, boolean tilgangTilKode7) {
+        if (tilgangTilKode6 && tilgangTilKode7) {
+            return "{}";
+        }
+        if (tilgangTilKode6) {
```

Comment:
is this correct, tilgangTilKode6 returns 7?

Reply:
Well it is a bit weird, but instead of returning a list of the kodes you are allowed to view, I instead retuned a list of codes you are not allowed to view. That is because I dont know the full list of codes

Reply:
aha, then maybe change name of the function

Reply:
changed it to a more understandable name 👍